docs/shrc.md

```diff
@@ -60,7 +60,7 @@ These cache utilities are discussed in more detail in the sections that follow.
 Class data sharing is enabled by default for bootstrap classes, unless your application is running in a container.
 Default behavior includes the following characteristics:
 
-- On Windows&reg;, the cache is created in the user's `C:\Users\<username>\AppData\Local\javasharedresources` directory. On z/OS&reg;, the default cache directory is `/tmp/javasharedresources`. On other systems, the cache is created in the `.cache/javasharedresources` directory in the user's home directory, unless the `groupAccess` parameter is specified, in which case it is created in `/tmp/javasharedresources`. Please do not set the home directory on a NFS mount or shared mount across systems or LPARs.
+- On Windows&reg;, the cache is created in the user's `C:\Users\<username>\AppData\Local\javasharedresources` directory. On z/OS&reg;, the default cache directory is `/tmp/javasharedresources`. On other systems, the cache is created in the `.cache/javasharedresources` directory in the user's home directory, unless the `groupAccess` parameter is specified, in which case it is created in `/tmp/javasharedresources`. Please do not set the home directory on an NFS mount or shared mount across systems or LPARs.
```

Comment:
Why "an NFS"? NFS starts with a consonant.

Reply:
We put "an" for an abbreviation, word, or number that begins with a vowel sound.